modules/boot-api/src/test/java/org/mule/runtime/module/boot/internal/DefaultMuleClassPathConfigTestCase.java

```diff
@@ -106,7 +111,9 @@ public void setUp() {
         createModuleLayerClassLoader(muleClassPathConfig.getOptURLs().toArray(new URL[muleClassPathConfig.getOptURLs().size()]),
                                      muleClassPathConfig.getMuleURLs().toArray(new URL[muleClassPathConfig.getMuleURLs().size()]),
                                      MULTI_LEVEL_URL_CLASSLOADER_FACTORY,
-                                     getSystemClassLoader());
+                                     new URLClassLoader(muleClassPathConfig.getResourceURLs()
```

Comment:
this is replicating the actual behaviour, instead of testing the productive code. If the productive code is broken, this test will still pass